trunk-recorder/main.cc

```diff
@@ -951,6 +951,14 @@ void current_system_status(TrunkMessage message, System *sys) {
   }
 }
 
+void current_system_sysid(TrunkMessage message, System *sys) {
+  if ((sys->get_system_type() == "p25") || (sys->get_system_type() != "conventionalP25")) {
```

Comment:
Should this be != "conventionalP25"? Since this is an OR, it seems like it would be true for every system type, except for "conventionalP25